### DISCUSSION TITLE
rancher plan.upgrade.cattle.io/system-agent-upgrader rke2

### DISCUSSION THREAD

**Discussion by frit0-rb** [rancher/system-upgrade-controller#309] [Q&A] [answered]
Good morning everyone,

I got two cluster inside my rancher, one create via Rancher and another one create outside directly with RKE2 and imported to rancher.

Im wondering why I have a labels sinside of the nodes calls plan.upgrade.cattle.io/system-agent-upgrader

I dont have any kind plan created or something, and I want to know if that is something I can delete inside of the labels.

Or maybe its something rancher is using to keep the connection alive pointing to another workloads or something else.

![image](https://github.com/rancher/system-upgrade-controller/assets/56691791/ead010fa-e190-4f63-bb19-64161869723f)

I see te Rancher cluster and thats labels don´t exists

Thanks in adavance

**Comment by brandond** [comment, marked as answer]
That plan is managed by Rancher. If you delete those labels, the system-agent-upgrader upgrade plan will run again. Don't muck about with it.

**Reply by frit0-rb** [reply]
Thanks @brandond
